src/main/java/seedu/exercise/model/ModelManager.java

```diff
@@ -229,11 +229,11 @@ public void completeSchedule(Schedule schedule) {
 
     /**
      * Returns an unmodifiable view of the list of {@code Exercise} backed by the internal list of
-     * {@code versionedExerciseBook}
+     * {@code exerciseBook}.
      */
     @Override
     public ObservableList<Exercise> getFilteredExerciseList() {
-        return filteredExercises;
+        return FXCollections.unmodifiableObservableList(filteredExercises);
```

Comment:
Why a sudden call to `FXCollections.unmodifiableObservableList`? A `FilteredList` is also `Unmodifiable`

Reply:
Aite. Noted!